src/runtime/runtime_avr.go

```diff
@@ -93,3 +95,12 @@ func abort() {
 		sleepWDT(WDT_PERIOD_2S)
 	}
 }
+
+//export __sync_lock_test_and_set_2
+func __sync_lock_test_and_set_2(ptr *uint16, value uint16) uint16 {
```

Comment:
Whatever the exact location, these two functions should _definitely_ be together in the same file.

Reply:
Yeah i think this was a mistake